keyboards/arisu/keymaps/default/keymap.c

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2019 Fate
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum arisu_layers {
+  _QWERTY,
+  _FN,
+  _ADJUST
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  FN,
+  ADJUST
+};
+
+// Tap Dance Declarations
+enum {
+    TD_LALT_LGUI = 0,
+    TD_RALT_RGUI
+};
+
+#define FN MO(_FN)
+#define ADJUST MO(_ADJUST)
+#define LALT_LG TD(TD_LALT_LGUI)
+#define RALT_RG TD(TD_RALT_RGUI)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = LAYOUT( /* Base */
+    KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_GRV,  KC_BSLS, KC_PGUP, \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want, but the purpose of them is actually to allow formatting long `#defines` over multiple lines, such as in the `LAYOUT_*` definitions.